ego/reference/remote-attestation.md

```diff
@@ -0,0 +1,36 @@
+# Remote Attestation
+
+## Requirements
+
+Make sure to install the following services to be able to use remote attestation:
```

Comment:
Maybe something along the lines of "Make sure you have access to the following services..." would be better, since a user won't have to install a PCCS if they are using e.g. an Azure confidential computing VM